### PR TITLE
Due to Button Size fixed some CSS issues there, This PR fixed that for Buttons

### PR DESCRIPTION
**Pull request by oumkale**
**Due to Button Size fixes some CSS were improper, This PR fix that **:

**Checklist**

* [ ] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [x] Have you added debug messages where necessary?
* [x] Have you added task comments where necessary?
* [x] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any?
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure?
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks?
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted?

**Special notes for your reviewer**:

### CODE REVIEW

File: litmus-portal/frontend/src/components/WelcomeModal/index.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const Welcomemodal: React.FC<WelcomemodalProps> = ({ isOpen }) => {
         open={open}
         onClose={handleClose}
         disableBackdropClick
-        disableEscapeKeyDown
+        disableEscapeKeyDown
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/WelcomeModal/index.tsx:33]: you can remove the unnecessary file changes like this trailing space...don't include these files

File: litmus-portal/frontend/src/components/WelcomeModal/styles.ts [hunk 2]
```diff
@@ -73,4 +73,4 @@ const useStyles = makeStyles((theme: Theme) => ({
     marginBottom: theme.spacing(1),
   },
 }));
-export default useStyles;
+export default useStyles;
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/WelcomeModal/styles.ts:76]: one new line was deleted

File: litmus-portal/frontend/src/components/WorkflowCluster/index.tsx [hunk 3]
```diff
@@ -100,4 +100,4 @@ const WorkflowCluster = () => {
   );
 };
 
-export default WorkflowCluster;
+export default WorkflowCluster;
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/WorkflowCluster/index.tsx:103]: same new line deleted

File: litmus-portal/frontend/src/components/WorkflowCluster/styles.ts [hunk 4]
```diff
@@ -42,17 +42,16 @@ const useStyles = makeStyles((theme: Theme) => ({
...
     flexDirection: 'row',
   },
   or: {
-    marginTop: theme.spacing(1),
+    marginTop: theme.spacing(1.5),
   },
 }));
 
-export default useStyles;
+export default useStyles;
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/WorkflowCluster/styles.ts:57]: new line deleted

Comment (arkajyotiMukherjee) [reply litmus-portal/frontend/src/components/WorkflowCluster/styles.ts:57]: are these don't by prettier?...generally new line is added at the end of the file

File: litmus-portal/frontend/src/containers/app/App.tsx [hunk 5]
```diff
@@ -22,7 +22,7 @@ interface RoutesProps {
 
 const Routes: React.FC<RoutesProps> = ({ userData }) => {
   const classes = useStyles();
-
+
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/containers/app/App.tsx:25]: empty space

File: litmus-portal/frontend/src/containers/app/App.tsx [hunk 6]
```diff
@@ -33,7 +33,7 @@ const Routes: React.FC<RoutesProps> = ({ userData }) => {
       </div>
     );
   }
-
+
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/containers/app/App.tsx:36]: empty space

File: litmus-portal/frontend/src/components/WorkflowCluster/index.tsx [hunk 7]
```diff
@@ -26,10 +25,8 @@ const WorkflowCluster = () => {
   const classes = useStyles();
   const [value, setValue] = React.useState('Experiment');
 
-  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setValue((event.target as HTMLInputElement).value);
-  };
-  const handleClick = () => {};
+  const handleChange = () => {};
+  const handleClick=()=>{}
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/WorkflowCluster/index.tsx:29]: why does these two lines have different formatting...are you not running prettier?

Comment (oumkale) [reply litmus-portal/frontend/src/components/WorkflowCluster/index.tsx:29]: prettier it is not working in installed it, Can you help me also npm run format showing err

**Comment by oumkale** [comment]
Things got fixed, @rajdas98 , @gdsoumya  @arkajyotiMukherjee ,  @S-ayanide , @Saranya-jena , @AmitKumarDas PTAL

**Pull request merged**